### DISCUSSION TITLE
Add a flag to hide the Delete Account button in the ProfileScreen

### DISCUSSION THREAD

**Discussion by WaliHassanKhan** [firebase/FirebaseUI-Flutter#397] [Ideas]
The delete account button should not be visible by default. 
I've read that we can create a custom widget for this. But i wonder if we simply add a flag to remove the delete account button?

**Comment by atreeon** [comment]
Hi Wali, did you solve this?

**Comment by WaliHassanKhan** [comment]
No, i have removed the page. 
I'll have to make my own profile screen.

**Reply by atreeon** [reply]
This page looks like there are some options for customisation of pages https://github.com/firebase/FirebaseUI-Flutter/blob/main/docs/firebase-ui-auth/providers/email.md
AuthFlowBuilder, FirebaseUIActions, EmailAuthListener etc
I'll be looking at this myself so please paste any solutions.
